mmdet3d/models/roi_heads/bbox_heads/multi_group_head.py

```diff
@@ -0,0 +1,880 @@
+import copy
+import numpy as np
+import torch
+from mmcv.cnn import ConvModule, build_conv_layer, kaiming_init
+from torch import nn
+
+from mmdet3d.core import circle_nms, xywhr2xyxyr
+from mmdet3d.models.utils import clip_sigmoid
+from mmdet3d.ops.iou3d.iou3d_utils import nms_gpu
+from mmdet.core import build_bbox_coder, multi_apply
+from mmdet.models import FeatureAdaption
+from mmdet.models.utils import gaussian_radius, gen_gaussian_target
+from ...builder import HEADS, build_loss
+
+
+class CenterPointFeatureAdaption(FeatureAdaption):
```

Comment:
The `CenterPointFeatureAdaption` is the same as [DCN layer](https://github.com/open-mmlab/mmcv/blob/master/mmcv/ops/deform_conv.py#L240), so we can directly use `DCN` layer by setting up config of dcn.